tests/components/bottom_bar.test.ts

```diff
@@ -256,6 +262,16 @@ describe("BottomBar component", () => {
       expect(document.activeElement).toEqual(sheetName);
     });
 
+    test("Double click on the sheet name in readonly mode doesn't make it editable", async () => {
+      model.updateMode("readonly");
+      await nextTick();
+      const sheetName = fixture.querySelector<HTMLElement>(".o-sheet-name")!;
+      expect(sheetName.getAttribute("contenteditable")).toEqual("false");
+      triggerMouseEvent(sheetName, "dblclick");
```

Comment:
if you end up with a conflit in futur versions, don't foret to adapt this line to use the helper 'doubleClick'